src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -2,12 +2,15 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
+using System.IO;
 using static Microsoft.Azure.WebJobs.Script.EnvironmentSettingNames;
 
 namespace Microsoft.Azure.WebJobs.Script
 {
     internal static class EnvironmentExtensions
     {
+        internal static string BaseDirectory { get; set; }
```

Comment:
Can you make a comment that this is for testing? I was trying to figure out where it got set :-)

Reply:
Funny... I had a comment and removed :)